tgstation.dme

```diff
@@ -2460,6 +2463,7 @@
 #include "code\modules\modular_computers\file_system\programs\ntmonitor.dm"
 #include "code\modules\modular_computers\file_system\programs\ntnrc_client.dm"
 #include "code\modules\modular_computers\file_system\programs\powermonitor.dm"
+#include "code\modules\modular_computers\file_system\programs\radar.dm"

```

Comment:
what's with these additions?

Reply:
Great question. I pulled from Summerball2020 so I have no clue where or how I picked that up. Ask around?